deploy/helm/sumologic/templates/events-configmap.yaml

```diff
@@ -10,4 +10,5 @@ data:
   fluent.conf: |-
     @include events.conf
   {{- (tpl (.Files.Glob "conf/events/*.conf").AsConfig .) | nindent 2 }}
+  {{- (tpl (.Files.Glob "conf/buffer.output.conf").AsConfig .) | nindent 2 }}
```

Comment:
where was this added? was it missed before?

Reply:
Yes, it was added in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/327